packages/lex-cli/src/codegen/client.ts

```diff
@@ -137,6 +162,27 @@ const indexTs = (
         name: ns.propName,
         type: ns.className,
       })
+
+      file
+        .addImportDeclaration({
+          moduleSpecifier: `./ns/${ns.propName}/index.js`,
```

Comment:
Does it make sense to use distinct file path for `client` and `server`, instead of `/ns/` for both ? I can see a future in which an app would generate both client and server definitions.